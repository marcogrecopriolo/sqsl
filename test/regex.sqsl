{ simple regex - populate }
display "output 10 to 50 separated by |";
output to "regexin.res";
for  i in 10, 20, 30, 40, 50
do
        display i, i+1, i+2 format delimited;
end for;
append to default;

input from "regexin.res" pattern regexp delimited by "\|";

{ more complex example }
display "output 10 to 50 separated by a*b";
output to "regexin.res";
for  i in 10, 20, 30, 40, 50
do
        display i, i+1, i+2, i+3 format full "%iab%iaab%ib%i";
end for;
append to default;

{ simple regex }
input from "regexin.res" pattern regexp delimited by "a*b";

{ split }
display "split by a*b";
display split "10ab11aab12b13" pattern regexp delimited by "a*b";
