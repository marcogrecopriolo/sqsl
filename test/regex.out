output 10 to 50 separated by |
10,11,12,
20,21,22,
30,31,32,
40,41,42,
50,51,52,
output 10 to 50 separated by a*b
10,11,12,13
20,21,22,23
30,31,32,33
40,41,42,43
50,51,52,53
split by a*b
10,11,12,13
